### PR TITLE
Translate Standardized Glossary page into Bahasa

### PR DESCRIPTION
**Pull request by habibrosyad**
Translate https://k8s.io/docs/reference/glossary into Bahasa as well as updating i18n to the latest upstream English updates.

/cc @girikuncoro @ariscahyadi

### CODE REVIEW

**Comment by habibrosyad** [comment]
Preview: https://deploy-preview-24180--kubernetes-io-master-staging.netlify.app/id/docs/reference/glossary

File: i18n/id.toml [hunk 1]
```diff
@@ -1,145 +1,227 @@
...
 
-[main_contribute]
-other = "Kontribusi"
-
-[main_edit_this_page]
-other = "Ubah halaman ini"
-
 [main_page_history]
-other ="Riwayat halaman"
+other ="Riwayat Halaman"
```
Comment (ariscahyadi) [inline i18n/id.toml:170]: nit!
```suggestion
other ="Riwayat laman"
```

File: i18n/id.toml [hunk 2]
```diff
@@ -1,145 +1,227 @@
...
+[post_create_issue]
+other = "Buat isu"
 
-[docs_label_i_am]
-other = "AKU ADALAH..."
+[prerequisites_heading]
+other = "Sebelum kamu memulai"
 
+[seealso_heading]
+other = "Lihat Juga"
```
Comment (ariscahyadi) [inline i18n/id.toml:197]: nit!
```suggestion
other = "Lihat juga"
```

File: i18n/id.toml [hunk 3]
```diff
@@ -1,145 +1,227 @@
...
-other = "Stack Overflow"
-[community_forum_name]
-other = "Forum"
-[community_events_calendar]
-other = "Kalender acara"
+[synopsis_heading]
+other = "Sinopsis"
+
+[thirdparty_message]
+other = """Bagian ini tertaut ke proyek-proyek pihak ketiga yang menyediakan fungsionalitas yang dibutuhkan oleh Kubernetes. Pencipta proyek Kubernetes tidak bertanggung jawab atas proyek-proyek tersebut. Halaman ini mengikuti <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">pedoman website CNCF</a> dengan membuat daftar proyek menurut abjad. Untuk menambakan proyek ke dalam daftar ini, bacalah <a href="/docs/contribute/style/content-guide/#third-party-content">panduan</a> sebelum mengirimkan perubahan."""
```
Comment (ariscahyadi) [inline i18n/id.toml:206]: nit!
```suggestion
other = """Bagian ini tertaut ke proyek-proyek pihak ketiga yang menyediakan fungsionalitas yang dibutuhkan oleh Kubernetes. Pencipta proyek Kubernetes tidak bertanggung jawab atas proyek-proyek tersebut. Laman ini mengikuti <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">pedoman website CNCF</a> dengan membuat daftar proyek menurut abjad. Untuk menambahkan proyek ke dalam daftar ini, bacalah <a href="/docs/contribute/style/content-guide/#third-party-content">panduan</a> sebelum mengirimkan perubahan."""
```

**Review by ariscahyadi — COMMENTED** [review]
Thanks @habibrosyad for your contribution!
For me, it seems OK, just some nits to address.
Let's wait @girikuncoro or @irvifa to approve!
/lgtm

**Comment by irvifa** [comment]
@ariscahyadi should we wait until the person fix this or should we merge it now?

**Comment by habibrosyad** [comment]
I'm going to push the update in a moment... 😄

**Comment by habibrosyad** [comment]
@ariscahyadi @irvifa updated...

**Comment by irvifa** [comment]
Let wait until the build is finished

**Comment by ariscahyadi** [comment]
I think it is ok!
/lgtm

**Pull request merged**